### PR TITLE
story 1

### PR DESCRIPTION
**Pull request by Shadow090509**

### CODE REVIEW

File: app/controllers/animals_controller.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  end
+end
+def destroy
+  animal = Animal.find(params[:id])
+  if animal.destroy
+    render json:animal
+  else
+    render json:animal.errors
+  end
+end
```
Comment (WalkerTexas-coder) [inline app/controllers/animals_controller.rb:38]: everything should be indented once over

File: app/controllers/animals_controller.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+private
+def animal_params
+  params.require(:animal).permit(:common_name, :scientific_binomial)
+end
```
Comment (WalkerTexas-coder) [inline app/controllers/animals_controller.rb:43]: indentation

**Review by WalkerTexas-coder — APPROVED** [review]
Looks good! Format your document and merge

File: app/controllers/animals_controller.rb (hunk 1 above)
Comment (Shadow090509) [reply app/controllers/animals_controller.rb:38]: what do you mean by indentation?

File: app/controllers/animals_controller.rb (hunk 2 above)
Comment (Shadow090509) [reply app/controllers/animals_controller.rb:43]: i'm not sure i understand

**Pull request merged**